student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(id, age, name: 'Unknown', parent_permission: true, classroom: nil)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] [Optional] [Highly recommended] If you implement the change I recommended above for the `id`, Please remove the id here from lines 4 and 5.